File: PHP/Lab-4/q4.php
<!-- Q4. Write a PHP program that shows even numbers from 1 to 20 using a while loop. -->
<?php
$i = 0;
while ($i <= 20) {
    if ($i % 2 == 0) {
        echo $i . "<br>";
    }
    $i++;
}
?>

File: Python/Lab-1/q4.py
# 4. Write a Python program to create a dictionary with 3 key-value pairs and print each key and its corresponding value.

student_marks = {
    "Ramesh" : 88,
    "Suresh" : 78,
    "Mahesh" : 85
}

# .items() returns a list of keys and values that is inserted in student,marks by foreach loop 
for student, marks in student_marks.items():
    print(f"Student: {student}, Marks: {marks}")